share/ci/pypicongpu_generator.py

```diff
@@ -144,19 +194,13 @@ def get_supported_versions(package_name: str, versions: List[str]) -> List[str]:
     global req_versions
     filtered_versions = []
 
+    requirement_file_names = sys.argv[1:]
+
     if not req_versions:
-        with open(sys.argv[1], "r", encoding="utf-8") as req_file:
-            parsed_req = pkg_resources.parse_requirements(req_file)
-            # ignore all lines, which cannot be parsed
-            # e.g. `-r extra/requirements.txt`
-            try:
-                for req in parsed_req:
-                    req_versions[req.project_name] = req
-            except Exception:
-                pass
+        req_versions = read_in_requirement_files(requirement_file_names)
 
     if package_name not in req_versions:
-        print(cs(f"ERROR: {package_name} is not defined in {sys.argv[1]}", "Red"))
+        print(cs(f"ERROR: {package_name} is not defined in {sys.argv[1]} or {sys.argv[2]}", "Red"))
```

Comment:
If I'm not wrong, the modification allows to handle N `requirements.txt`. Therefore the error messages should be modified.

Reply:
fixed